docs/configuration/configuration.md

```diff
@@ -1341,6 +1345,53 @@ for a detailed example of configuring Prometheus for Kubernetes.
 You may wish to check out the 3rd party [Prometheus Operator](https://github.com/coreos/prometheus-operator),
 which automates the Prometheus setup on top of Kubernetes.
 
+### `<lightsail_sd_config>`
+
+Lightsail SD configurations allow retrieving scrape targets from AWS Lightsail
```

Comment:
Can you add a link to lightsail?

Reply:
https://github.com/prometheus/prometheus/pull/8693/commits/3db49731a807b670e91fc84ce6f402f19cb8a0df